contrib/templates/experimental/scala-job/template/project/plugins.sbt.tmpl

```diff
@@ -0,0 +1 @@
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.0.0")
```

Comment:
File should have a comment at the top, also explaining what the `project/` folder is about. (A separate README for that seems like overkill.)